src/save_audio.rs

```diff
@@ -90,6 +93,8 @@ pub fn save_audio_file_to_disk(mut audio_bytes: Vec<u8>, device_config: DeviceCo
                 let sample_rate = LittleEndian::read_u16(&audio_bytes[10..12]) as u32;
```

Comment:
Round to known "real" sample rates, or just set to 48kHz here since we know we're not setting it to anything else at the moment.